script/storybook-deploy.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+if [[ $( git diff --name-only HEAD~1..HEAD webpack/ package.json | wc -l ) -ne 0 ]]; then
+  REPO_SLUG_ARRAY=(${TRAVIS_REPO_SLUG//\// })
+  REPO_OWNER=${REPO_SLUG_ARRAY[0]}
+  REPO_NAME=${REPO_SLUG_ARRAY[1]}
+  DEPLOY_PATH=storybook-static
+
+
+  DEPLOY_SUBDOMAIN_UNFORMATTED_LIST=()
+  if [ "$TRAVIS_PULL_REQUEST" != "false" ]
```

Comment:
Nice!
Can we add one to develop branch as well? So we always have the latest develop version deployed.

Reply:
Don't we already have that at http://theforeman.github.io/foreman/?

Reply:
ATM I push it manually every now and then...

Reply:
I think the ask was, should we remove it from github.io or add redirect when we merge this. 

[Previously](https://community.theforeman.org/t/improving-the-ui-documentation/9676/5) it has been discussed it should be pushed as part of CI and it should be linked from developer hand book, maybe with the new url, we should update the hand book.

Reply:
I don't mind, this was a stop gap as noone else did it, nevertheless, this pr doesn't not address that usage case, having said that, the fact there is no link from https://theforeman.org/handbook.html#JavaScript is pretty surprising (regardless of how its published)...

Reply:
https://docs.travis-ci.com/user/deployment/surge/ is probably much easier to use for post-merge builds.